.nvmrc

```diff
@@ -1 +1 @@
-v14.19.0
+stable
```

Comment:
Will this affect our package.json scripts or is everything we're using already built on / compatible with whatever stable is currently?

Reply:
20 is current stable. 14 is defunct. If anything they should be more comparable with 20. This won't affect anything in prod as that's all browser code. Usually changing this is very safe for these situations cause the package managers are relatively current. V18 would be a little safer.

Reply:
I'm a PHP dude, so my reference is to that and composer. _IF_ changing the node version also changes what package versions get installed, then we should fix the version of node to whatever and rebuild the plugin assets. 

(This would _not_ be a breaking change I believe, since its only used for building)

Reply:
I don't believe it does. `package.json` determines those in node.